code/obj/item/cigarette.dm

```diff
@@ -2,6 +2,136 @@
 /* ==================================================== */
 /* -------------------- Cigarettes -------------------- */
 /* ==================================================== */
+/obj/item/clothing/mask/cig_bundle
+	name = "bundle of cigarettes"
+	icon = 'icons/obj/items/cigarettes.dmi'
+	wear_image_icon = 'icons/mob/mask.dmi'
+	icon_state = "cig_bundle-2"
+	uses_multiple_icon_states = 1
+	item_state = "cig_bundle"
+	var/list/cig_list = list()
+	var/const/max_cigs = 21
+	var/on = 0
+	cant_self_remove = 1
+
+	New(var/obj/item/clothing/mask/cigarette/cig1, var/obj/item/clothing/mask/cigarette/cig2)
+		..()
+		cig1.set_loc(src)
```

Comment:
Check that `cig1` and `cig2` actually exist, and maybe provide some alternative effect if they aren't (just randomly add some amount of cigs to the bundle?). An admin could spawn these, for instance, and these arguments wouldn't be provided.

Reply:
I can add that. I made these with the intent that they be created and used in a singular, very specific circumstance so I didn't consider that an admin might want to spawn one. 

The object should pretty much only exist in the mask slot of a Human's HuD and nowhere else. It breaks down into its component cigaretted when dropped and can't be removed from the mask slot without removing each cigarette.